rules/slack_rules/slack_app_access_expanded.py

```diff
@@ -27,7 +31,7 @@ def alert_context(event):
     prv_scopes = event.deep_get("details", "previous_scopes", default=[])
 
     context["scopes_added"] = [x for x in new_scopes if x not in prv_scopes]
-    context["scoped_removed"] = [x for x in prv_scopes if x not in new_scopes]
+    context["scopes_removed"] = [x for x in prv_scopes if x not in new_scopes]
```

Comment:
```suggestion
    # Diff previous and new scopes
    changes = get_scope_changes(event)
    context["scopes_added"] = changes["added"]
    context["scopes_removed"] = changes["removed"]

    return context
```
Again, using a new function since now we need this logic in 2 places